dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ElasticManager.java

```diff
@@ -182,6 +184,19 @@ public String updateDocumentValueFromLabels(Entry entry) {
         return builder.toString();
     }
 
+    public void bulkUpsert(List<Entry> entries) {
+        try (RestClient restClient = RestClient.builder(new HttpHost(ElasticManager.hostname, ElasticManager.port, "http")).build()) {
+            String newlineDJSON = getNDJSON(entries);
+            HttpEntity bulkEntity = new NStringEntity(newlineDJSON, ContentType.APPLICATION_JSON);
+            org.elasticsearch.client.Response post = restClient.performRequest("POST", "/entry/_bulk", Collections.emptyMap(), bulkEntity);
+            if (post.getStatusLine().getStatusCode() != HttpStatus.SC_OK) {
+                throw new CustomWebApplicationException("Could not submit index to elastic search", HttpStatus.SC_INTERNAL_SERVER_ERROR);
```

Comment:
Will this return a 500 if elastic search is not available?
I believe that the strategy for tool refresh was that we would not kill the whole refresh if elastic search was not available.

Reply:
This won't, but the line under it will.  Which I've just changed.